test/plugins/link.js

```diff
@@ -26,7 +29,7 @@ describe('Link plugin', function() {
     });
 
     this.irc.createMessage({
-      message: 'http://localhost:9002/basic'
+      message: 'http://127.0.0.1:9002/basic'
```

Comment:
The reason is that we do not want to prefetch links that point to `localhost`. But that's actually a flaw: if we don't want that, we should also prevent `127.0.0.1`, `[::1]`, ... So here again, the problem should be solved at a deeper level :(